iroha/src/lib.rs

```diff
@@ -296,6 +304,26 @@ impl Iroha {
         );
         Ok(())
     }
+
+    /// To wait for active peers(this is provisional code)
+    async fn wait_for_active_peers(&self, sorted_peers: &[PeerId]) -> Result<(), String> {
+        log::info!("Waiting for active peers");
+        for peer in sorted_peers.iter() {
+            let url = httpUri::Builder::new()
+                .scheme("http")
+                .authority(peer.address.as_str())
+                .path_and_query(uri::HEALTH_URI)
+                .build()
+                .expect(&format!("invalid peer address {}", peer.address.as_str()));
```

Comment:
It would be better to start error messages with a big letter. E.g. `Invalid peer ... `
Please also correct for the other error messages below.